Formula/p/pixi.rb

```diff
@@ -34,6 +34,10 @@ class Pixi < Formula
   end
 
   def install
+    if build.head?
+      ENV["PIXI_VERSION"] = Utils.safe_popen_read("git", "describe", "--tags").chomp.delete_prefix("v")
```

Comment:
```suggestion
      system "git", "fetch", "--tags"
      ENV["PIXI_VERSION"] = Utils.safe_popen_read("git", "describe", "--tags").chomp.delete_prefix("v")
```

https://github.com/Homebrew/homebrew-core/pull/195551#discussion_r1816719933

Reply:
if you do head installation, it would be always fresh fetch, so you probably dont need another git fetch?

Reply:
yeah seems to work now without the fetching 🤔 no idea why it didn't work before...